pynitrokey/start/upgrade_by_passwd.py

```diff
@@ -158,80 +169,103 @@ def main(wait_e, keyno, passwd, data_regnual, data_upgrade, skip_bootloader, ver
         signed_bytes = rsa.integer_to_bytes_256(signed)
         gnuk.cmd_external_authenticate(keyno, signed_bytes)
         gnuk.stop_gnuk()
+
         mem_info = gnuk.mem_info()
+        # @todo: use global verbosity
         if verbosity:
             local_print("%08x:%08x" % mem_info)
 
-        local_print('*** Running update. Do NOT remove the device from the USB slot, '
-                    'until further notice.')
+        local_print("Running update!",
+                    "Do NOT remove the device from the USB slot, until further notice",
+                    "Downloading flash upgrade program...")
 
```

Comment:
Asterisks `***` were meant to focus user on this specific lines, as many of them are printed out, and not all are important. Ideally the ones without this marker should be silenced completely to not confuse, and only output them in the log.

Reply:
mainly done for some more output consistency, should decide on some "scheme" for user outputs, next PR(s)